Logo/bin/generate-assets.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+wd=`pwd`
+destdir="autogenerated"
+pngwidths="300 600 1200"
```

Comment:
Just 600px and 1200px is enough for the PNGs, there are already a lot of generated files.

Reply:
Ok, done on my local version. Don't know if it is right to write this here, but is for tracking changes :-)

Reply:
No problem, next time you can just click the "Resolve conversation" button in this comment thread. I'll do that now.